### DISCUSSION TITLE
How to increase the width margins of the about.md

### DISCUSSION THREAD

**Discussion by Wongolini** [academicpages/academicpages.github.io#2365] [Q&A]
Hi I noticed that the margins in the other tabs (blog, talks, teaching) have wider margins where the text and images can go to the farther edges to the right. I was wondering how to change this for the about home page to make the text go to the right more.

<img width="1503" alt="image" src="https://github.com/user-attachments/assets/2da99f8e-66ad-45f0-bd79-d07dcecfbfdc">

versus

<img width="1503" alt="image" src="https://github.com/user-attachments/assets/05c516b7-2489-4a04-8cd6-3da25b05f3a4">

**Comment by jakesong15** [comment]
I had this issue as well and managed to find a work-around solution for this (YMMV):

For pages where you want to **remove** the empty space on the right, begin the page files in your _pages directory with **layout: archive**
For pages where you want to **insert** the empty space on the right, begin the page files in your _pages directory with **layout: single**

**Reply by Wongolini** [reply]
Did you put in under the 
`
---
permalink: /
title: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
`
?

**Reply by jakesong15** [reply]
I put it on the very first line, above permalink but below the ---. Worked fine for me!
